File: Python/uberPool.py
from car import Car

class UberPool(Car):
    brand = str
    model = str

    def __init__(self, license, driver, brand, model):
        super().__init__(license,driver)
        self.brand = brand
        self.model = model

    def printdataCar(self):
        print("La licencia es: " + self.license + ". \n" + "El conductor es: " + self.driver.name + ". \n" + "El documento es: " + self.driver.document + ". \n" + "La marca es: " + self.brand + ". \n" + "El modelo es: " + self.model + ".")
        #super().printdataCar()

File: PHP/uberPool.php
<?php
require_once('car.php');
class UberPool extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license,$driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function printDataCar(){
        echo 'La licencia es: '
        . $this->license .". \n"
        .'El conductor es: '. $this->driver->name .". \n"
        .'El documento es: ' . $this->driver->document .". \n"
        .'La marca de auto es: '. $this->brand .". \n"
        .'El modelo de auto es: ' . $this->model .". \n";
        
    }

}
?>